creator-node/src/routes/healthCheck.js

```diff
@@ -6,6 +6,12 @@ const versionInfo = require('../../.version.json')
 module.exports = function (app) {
   /** @dev TODO - Explore checking more than just DB (ex. IPFS) */
   app.get('/health_check', handleResponse(async (req, res) => {
+    // for now we just check db connectivity. In future, this could / should check other
```

Comment:
should we remove comment about db connectivity?

Reply:
thank you. fixing